docker/readme.md

```diff
@@ -0,0 +1,54 @@
+Docker Script Documentation
+===========================
+
+The files in this folder are intended to be mainly used by PyNE maintainers,
+they provide an "easy" mechanism to build and push the various containers
+required for the multiple configurations tested by the PyNE CI.
+
+This folder contains:
+
+- A custom Dockerfile: "ubuntu_18.04-dev.dockerfile", allowing the user 
+to build docker containers with a custom set of
+  dependencies required by the CI. The configuration of the Docker containers
+  can be specified using the `--build-args <flag>` argument. The different flags are:
+    
+    - `build_moab=YES` add MOAB in the docker container
+    - `build_pymoab=YES` install MOAB with pymoab
+    - `build_dagmc=YES` add DAGMC 
+    - `build_pyne=NO` do not build PyNE, only add the PyNE dependencies to the container
+    - `py_version=X.Y` specify the python version to install (options are 2.7 or
+      3.6)
+Note, if using multiple build arguments `--build-args` has to be repeted, i.e. :
+to request DAGMC and only dependencies with Python 2.7, the docker build arguments will be: 
+`--build-args build_dagmc=YES --build-args build_pyne=NO --build-args=py_version=2.7` 
```

Comment:
Does it look better @katyhuff ? 
(I also added an example...)

(thx for the review and the English corrections!)